apps/storybook-react-native/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@metamask/storybook-react-native",
+  "version": "0.0.0",
```

Comment:
Adding versions back here as the `prepare-preview-builds` script updates the workspace names. This prevents `Workspace not found` errors with Yarn workspaces, which look for specific version names when creating new preview workspace names. These packages still won’t be published as preview builds. We could potentially update the script to skip version updates in preview, but this should be fine as it shouldn’t affect the process.

![Screenshot 2024-11-06 at 1 47 21 PM](https://github.com/user-attachments/assets/09f00192-f665-4fae-aa32-3651bf6b7a6d)

[See thread](https://consensys.slack.com/archives/C01V1L10W2E/p1730932044619759?thread_ts=1730922932.508419&cid=C01V1L10W2E)